src/core/models/accounts.ts

```diff
@@ -0,0 +1,24 @@
+import { Schema, model } from "mongoose";
+
+const AccountSchema = new Schema({
+   user:{
+       type: Schema.Types.ObjectId,
+       ref: 'User'
+   },
+   //This field holds the profit made every quarter plus initial balance
+   current_balance: {
+    type: Float64Array,
```

Comment:
Why `Float64Array` instead of `number`?

Reply:
Chatgpt ba 😂😭

Reply:
i just wanted to be accurate 
if it dollars or if it naira 
superisingly this was me

Reply:
Interesting. No need for that, though; everything we are doing is in Naira. We can convert it to dollars or any other currency once we have fetched it from the db